fw/tls.c

```diff
@@ -817,8 +818,8 @@ tfw_tls_conn_send(TfwConn *c, TfwMsg *msg)
 	 * We can not send the alert on conn_drop hook, because the hook
```

Comment:
Please write a comment above the `ss_send()` that the actual message transmission may happen on another CPU, so after `ss_send()` `msg` can not be user. Otherwise we can step the same issue in future. Also `tfw_connection_send()` must be commented that it's unsafe to reference `msg` after the call and reference `tfw_tls_conn_send()`.